src/rpcserver.cpp

```diff
@@ -330,6 +330,7 @@ static const CRPCCommand vRPCCommands[] =
     { "getactivecrowdsales_MP", &getactivecrowdsales_MP, false,     false,      true },
 #if 0
     { "trade_MP",               &trade_MP,               false,     false,      true },
+    { "getsto_MP",              &getsto_MP,              false,     false,      true },
```

Comment:
Wondering about this one.. see: https://github.com/mastercoin-MSC/mastercore/blob/mscore-0.0.9/src/rpcserver.cpp#L341